content/podcast/2024/ep29_Nextflow-2023-2024.md

```diff
@@ -0,0 +1,53 @@
+title=Nextflow 2023 - 2024
+episode=29
+description=We recap of some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
```

Comment:
"We recap of some of the biggest features" 

extra 'of' snuck in there

Reply:
```suggestion
description=We recap some of the biggest features to be added to Nextflow in 2023 and take a look at some of the things coming in 2024.
```